dir.props

```diff
@@ -69,6 +69,8 @@
     <Platform>AnyCPU</Platform>
     <OutputType>Library</OutputType>
     <RunApiCompat>true</RunApiCompat>
+    <!-- Default to using the open public key -->
+    <AssemblyOriginatorKeyFile>$(ToolsDir)Open.snk</AssemblyOriginatorKeyFile>
```

Comment:
could you please check this doesn't change which key is used for the xamarin csproj's (e.g. System.Net.Http)?